### ISSUE TITLE
[Enhancement] Compatibility with Treecapitator

### ISSUE THREAD

**Issue by AlexPlus117** [SlimeKnights/IguanaTweaksTConstruct#192]
Hi boni, I want to suggest you to add a compatibility between Iguana Tinker Tweaks and Treecapitator so that if you cut a tree, the axe with which you've cut it increases its own xp according to the pieces of wood cut. Is it workable?

**Comment by bonii-xx** [comment]
That's out of my hands. If it doesn't then treecapitator doesn't use the tool to break the blocks.

**Comment by squeek502** [comment]
Yeah, Treecapitator never calls `onBlockStartBreak`, but it [does call `onBlockDestroyed`](https://github.com/bspkrs/Treecapitator/blob/1.7.10/src/main/java/bspkrs/treecapitator/Treecapitator.java#L710). ITT uses `onBlockStartBreak`.

**Comment by AlexPlus117** [comment]
So is it IMPOSSIBLE to do?

**Comment by squeek502** [comment]
Not impossible, just would make the most sense for Treecapitator to fix it. Unfortunately, I'm not sure if Treecapitator is actively maintained; I've [submitted a pull request that fixes a similar incompatibility](https://github.com/bspkrs/Treecapitator/pull/33) and it's gotten no action.

**Comment by AlexPlus117** [comment]
I see...we'll have to wait a bspkrs's reply.
